### PR TITLE
Em oh oh en, that spells brain damage lines!

### PR DESCRIPTION
**Pull request by Iamgoofball**
:cl: Tom Cullen
experimental: M-O-O-N, that spells changelog!
/:cl:

### CODE REVIEW

File: strings/brain_damage_lines.json [hunk 1]
```diff
@@ -63,6 +63,7 @@
         "@pick(semicolon)wtf??????????? @pick(bug)",
         "@pick(semicolon)i ran into the supermattre ten i dsappeard @pick(bug)",
         "DON'T EVER TUCH ME",
+		"EM OH OH EN, that speels @pick(the_stand)"
```
Comment (Iamgoofball) [inline strings/brain_damage_lines.json]: fucking formatting

Comment (Iamgoofball) [reply strings/brain_damage_lines.json]: is there a reason the standard for jsons is spaces versus tabs

**Comment by Iamgoofball** [comment]
@Cyberboss bot tagging for brain damage lines appears to be busted

**Comment by calzilla1** [comment]
What is this meme

**Comment by Iamgoofball** [comment]
@calzilla1 read more books

**Comment by Sartorius7** [comment]
It's a Stephen King reference.

**Comment by Iamgoofball** [comment]
@ShizCalev it's a joke about The Stand, not bait.

Tom Cullen is a character in that who is dumb as hell, implied due to abuse from his parents and a knock on the head, one of his common lines is "M-O-O-N, that spells *something relevant*".

**Comment by Iamgoofball** [comment]
i need to read the stand again, it's a good book

**Comment by optimumtact** [comment]
Whatever label we had for braindamage pr's is gone

**Comment by Iamgoofball** [comment]
Bot is supposed to apply it manually

**Comment by Cyberboss** [comment]
>Whatever label we had for braindamage pr's is gone

**Comment by lzimann** [comment]
yet another low effort brain damage pr

**Comment by Sartorius7** [comment]
It's a reference to a book do you even read

**Comment by lzimann** [comment]
it could reference the fucking bible and i would still not give a fuck

**Comment by Iamgoofball** [comment]
@KorPhaeron can you do something about this

**Comment by Grazyn** [comment]
\>literally a reference to an actual brain damage line used by a brain damaged character in a popular novel
\>uses randomized stuff to make it less repetitive every time it's uttered
\>dismissed as low effort

Remind me what's the point of having burned-out maintainers again?

**Comment by lzimann** [comment]
Oh wow, such a burned out maintainer stopping yet another brain damage line pr, someone stop this madman, otherwise people that never even contributed to anything will get mad!

**Comment by Bawhoppen** [comment]
Sing it with me, "MORE BRAIN DAMAGE LINES AREN'T FUNNY!~"

**Comment by Grazyn** [comment]
Because hearing the same lines from decades old /tg/ memes is ~so much fun!~

3 new lines added since June, someone stop all this bloat before with drown in pop culture references

**Comment by Bawhoppen** [comment]
No brain damage lines are funny
don't you get it

**Comment by Grazyn** [comment]
new brain damage line: "Brane damedge layns are funneh"

**Comment by Bawhoppen** [comment]
Actually that one would be kinda funny

**Comment by XDTM** [comment]
If my rework goes through all the lines will go into a single trauma which makes them less relevant

**Comment by Iamgoofball** [comment]
I'd ping lzimann and ask if he thinks all comedy is low effort unless it's a dank tee gee may may but he blocked me because his snowflake irc girlfriend got banned :(

**Comment by KorPhaeron** [comment]
I never read The Stand sorry

**Comment by KorPhaeron** [comment]
We should let XDTM finish his rework before we add more lines anyway

**Comment by Iamgoofball** [comment]
Go read it it's good

**Comment by Iamgoofball** [comment]
That's a good idea

**Comment by lzimann** [comment]
What are you even talking about? I never blocked you.

**Comment by Iamgoofball** [comment]
Fucking github

**Pull request closed without merging**